instrumentation/jmx-metrics/javaagent/camel.md

```diff
@@ -0,0 +1,54 @@
+# Camel Metrics
+
+Here is the list of metrics based on MBeans exposed by Camel.
+
+| Metric Name                                            | Type    | Attributes                             | Description                                                                          |
+|--------------------------------------------------------|---------|----------------------------------------|--------------------------------------------------------------------------------------|
+| org.apache.camel.context.exchanges.completed           | Counter | context, camelVersion                  | Exchanges Completed grouped by Camel Context                                         |
+| org.apache.camel.context.exchanges.failed              | Counter | context, camelVersion                  | Exchanges Failed grouped by Camel Context                                            |
+| org.apache.camel.context.exchanges.inflight            | Counter | context, camelVersion                  | Exchanges Inflight grouped by Camel Context                                          |
```

Comment:
should be UpDownCounter?

Reply:
yeah, it's a gauge

Reply:
have a look at https://opentelemetry.io/docs/specs/otel/metrics/supplementary-guidelines/#additive-property